src/signal.cr

```diff
@@ -191,7 +191,7 @@ module Crystal::Signal
 
   alias Handler = ::Signal ->
 
-  @@pipe = IO.pipe(read_blocking: false, write_blocking: true)
+  private class_getter(pipe) { IO.pipe(read_blocking: false, write_blocking: true) }
```

Comment:
Previously I was trying to import "signal.cr" and make all methods no-op. But it was messy and I reverted to not import signal at all (just like win32). This change was reverted but it was required because wasm doesn't have `IO.pipe`.